src/backend/.config/flag.json

```diff
@@ -0,0 +1,6 @@
+{
+  "feature_flags": {
+    "v1": true,
+    "v2": false
+  }
```

Comment:
Are we binding these values to ENV variables? Or is that meant to come later?

Reply:
I was thinking moreover of using more descriptive names like PIA_INTAKE_FORM_FLAG so that we can map functionality to a name. Thoughts?

Reply:
change to a more descriptive flag name

Reply:
`PIA_INTAKE_FORM_FLAG` sounds much better to me too